snap/snapcraft.yaml

```diff
@@ -142,7 +142,6 @@ parts:
     - git
     - golang
     - libsystemd-dev
-    source: .
```

Comment:
I would leave `source: .` here to make this future proof (implicit sources are not a thing in snapcraft 3.0 when using bases)